sdk/python/feast/infra/online_stores/datastore.py

```diff
@@ -387,6 +387,19 @@ def from_infra_object_proto(infra_object_proto: InfraObjectProto) -> Any:
 
         return datastore_table
 
+    @staticmethod
+    def from_proto(datastore_table_proto: DatastoreTableProto) -> Any:
+        datastore_table = DatastoreTable(
+            project=datastore_table_proto.project, name=datastore_table_proto.name,
+        )
+
+        if datastore_table_proto.HasField("project_id"):
```

Comment:
I think this will always be true since we're using proto3 & it auto populates these with the default values (https://developers.google.com/protocol-buffers/docs/reference/python-generated#singular-fields-proto3)

Reply:
ah so the `project_id` field is actually a message field I believe?

`google.protobuf.StringValue project_id = 3;`

so I think this should work

Reply:
Got it. Maybe add a comment that we're using StringValue? since proto3 by default makes string "".